canonical-web-and-design/snapcraft.io
# Simplify getprogressivestate

## Done

- Removed `revision` from `getProgressiveState` selector
- Tidied up code that calls `getProgressiveState`

## Issue / Card

Fixes #2345

## QA

- Pull the branch
- Run the site using the command `./run`
- View the site locally in your web browser at: http://0.0.0.0:8004/snap_name/releases
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Play with progressive releases, ensure that they work as expected
